app/components/DataTable/components/CivicCellRenderer/index.tsx

```diff
@@ -3,33 +3,88 @@ import React, { useState, useEffect } from 'react';
 import NewTabLink from '@/components/NewTabLink';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { ICellRendererParams } from '@ag-grid-community/core';
+import { IconButton, Menu, MenuItem } from '@material-ui/core';
+import { OpenInNew } from '@material-ui/icons';
 
 type CivicCellRendererProps = ICellRendererParams['data'];
 
 const CivicCellRenderer = ({
   data,
 }: CivicCellRendererProps): JSX.Element => {
+  const {
+    externalStatementId,
+    externalSource,
+  } = data;
```

Comment:
You can move this destructure into the props like:

```
const CivicCellRenderer = ({
  data: { externalStatementId, externalSource },
}: CivicCellRendererProps): JSX.Element => {
```

Reply:
I could, but this I feel like it's cleaner?

Reply:
Up to you then if you feel that way